src/data/auth/projectServices.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  ProjectsData,
+  NewProject,
+  DeleteProject,
+  Project,
+} from '../../types/project';
+import { http } from '../http/http';
+
+export const getAllProjects = async (): Promise<ProjectsData> => {
```

Comment:
Why do you prefer return `ProjectsData` instead of `Project[]`

Reply:
I can't explain it. Is my version not correct?